waffle_hub/hub/hub.py

```diff
@@ -402,6 +404,51 @@ def get_hub_list(cls, root_dir: str = None) -> list[str]:
                     hub_name_list.append(hub_dir.name)
         return hub_name_list
 
+    @classmethod
+    def import_waffle(cls, name: str, waffle_file: str, root_dir: str = None) -> "Hub":
```

Comment:
Using `from_waffle_file` would be better for the consistency among other class methods.